website/settings/local-dist.py

```diff
@@ -22,6 +22,11 @@
 #WATERBUTLER_URL = 'http://localhost:7777'
 #WATERBUTLER_INTERNAL_URL = WATERBUTLER_URL
 
+LIVE_RELOAD_DOMAIN = 'http://localhost:4200'
```

Comment:
With Retraction Watch and Registries, we're very close to having multiple `EXTERNAL_EMBER_APPS`.

How will this handle that scenario- do we just advise developers to change the `LIVE_RELOAD_DOMAIN` setting based on their current project?

(Documenting `EXTERNAL_EMBER_APPS` is outside scope of this PR, but it's worth thinking ahead about this feature. It would be great if we could avoid having to break dev setups that depend on this feature later)